changelog.md

```diff
@@ -1,3 +1,138 @@
+ <a name="5.2.0"></a>
+# 5.2.0 (2019-07-24)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v5.1.0...v5.2.0)
+
+## New Contributors!
+Thanks to @AngeloGulina, @jamesgeorge007, @johnemau, @kaushikchaubal, @mitchellsimoens, @NickolasBenakis, @noelyoo, @PatOnTheBack, @rishichawda, and @Remexllee for their first contributions!
+
+## New Audits
+
+* add third party entity summary ([#9067](https://github.com/googlechrome/lighthouse/pull/9067))
+* add Total Blocking Time metric to JSON output ([#8975](https://github.com/googlechrome/lighthouse/pull/8975), [#9409](https://github.com/googlechrome/lighthouse/pull/9409))
+
+## Core
+
+* driver: security errors are no longer a fatal or pageload error ([#8865](https://github.com/googlechrome/lighthouse/pull/8865))
+* support saving and loading errors in artifacts ([#9397](https://github.com/googlechrome/lighthouse/pull/9397))
+* config: assert all audit `requiredArtifacts` will be gathered ([#9284](https://github.com/googlechrome/lighthouse/pull/9284))
+* `domstats`: support an empty html body ([#9340](https://github.com/googlechrome/lighthouse/pull/9340))
+* `font-display`: limit false positives ([#9148](https://github.com/googlechrome/lighthouse/pull/9148))
+* `font-size`: calculate accurate line/column for styles ([#9356](https://github.com/googlechrome/lighthouse/pull/9356))
+* `font-size`: comment why source 'Other' occurs ([#9363](https://github.com/googlechrome/lighthouse/pull/9363))
+* localize invalid-URL error message ([#9334](https://github.com/googlechrome/lighthouse/pull/9334))
+* `gather-runner`: add `PageLoadError` base artifact ([#9236](https://github.com/googlechrome/lighthouse/pull/9236))
+* `gather-runner`: don't save trace on pass with `pageLoadError` ([#9198](https://github.com/googlechrome/lighthouse/pull/9198))
+* `gather-runner`: treat Chrome interstitials as `pageLoadErrors` ([#9176](https://github.com/googlechrome/lighthouse/pull/9176))
+* `gather-runner`: always reset scroll position ([#9060](https://github.com/googlechrome/lighthouse/pull/9060))
+* `gather-runner`: convert `PAGE_HUNG` to non-fatal `runtimeError` ([#9121](https://github.com/googlechrome/lighthouse/pull/9121))
+* `manifest-parser`: handle blob manifests ([#9088](https://github.com/googlechrome/lighthouse/pull/9088))
+* `page-functions`: don't try to clone a ShadowRoot ([#9079](https://github.com/googlechrome/lighthouse/pull/9079))
+* perf: speed up `tap-target`'s `isVisible()` ([#9056](https://github.com/googlechrome/lighthouse/pull/9056))
+* tracehouse: allow missing FCP ([#9174](https://github.com/googlechrome/lighthouse/pull/9174))
+* tracehouse: move and organize files in `lib/tracehouse/` ([9f80524](https://github.com/googlechrome/lighthouse/commit/9f80524), [1af2edd](https://github.com/googlechrome/lighthouse/commit/1af2edd), [7ba6776](https://github.com/googlechrome/lighthouse/commit/7ba6776), [ddff3d6](https://github.com/googlechrome/lighthouse/commit/ddff3d6))
```

Comment:
I wonder if this should be called out at all more if folks were relying on the path to do their own trace parsing?

Reply:
> I wonder if this should be called out at all more if folks were relying on the path to do their own trace parsing?

done